src/crypto/BUILD.gn

```diff
@@ -119,7 +119,8 @@ if (chip_crypto == "openssl") {
     sources = [ "CHIPCryptoPALPSA.cpp" ]
     public_deps = [ ":public_headers" ]
 
-    external_mbedtls = current_os == "zephyr"
+    external_mbedtls = current_os == "zephyr" || current_os == "cmsis-rtos"
```

Comment:
I think we can provide the same improvement here as is in mbedtls crypto (line 111)
```
if (!chip_external_mbedtls) {
      public_deps += [ "${mbedtls_root}:mbedtls" ]
 }
```
Zephyr platform has already set the `chip_external_mbedtls`. OIS platform also.